03_launch_mgmt_cluster.sh

```diff
@@ -551,12 +551,75 @@ cat <<EOF > tilt-settings.json
   }
 }
 EOF
+sed -i 's/yaml = str(kustomizesub(context + "\/config"))/yaml = str(kustomizesub(context + "\/config\/tls"))/' Tiltfile
 make kind-reset
 kind create cluster --name capm3 --image="kindest/node:${KUBERNETES_VERSION}"
 kubectl create namespace "${NAMESPACE}"
+kubectl create namespace baremetal-operator-system
```

Comment:
ns can be parameterized? `IRONIC_NAMESPACE`

Reply:
done